src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,77 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    public static final String SUPPLY = "supply,";
+    public static final String BUY = "buy,";
+    public static final String RESULT = "result,";
+    public static final String DELIMETER = ",";
+    private StringBuilder stringBuilder = new StringBuilder();
+    private int buy = 0;
+    private int supply = 0;
```

Comment:
as described in [common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/builder-file/work-with-file#local-variables), let's not make these variables class fields